### PR TITLE
Add Fieldlist example.

### PR DESCRIPTION
**Pull request by ghost**
Super basic example. May want to dynamically update the output when removing / adding fields?

### CODE REVIEW

File: example/src/controllers/examples/subsections/fieldlist/code.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    },
+
+    getInitialState() {
+        return {
+            uniqueTagId: 0,
+            pets: []
+        };
+    },
+
+    abandonPet(tagId) {
```
Comment (uttrasey) [inline example/src/controllers/examples/subsections/fieldlist/code.js]: :cry:

Comment (kaw2k) [inline example/src/controllers/examples/subsections/fieldlist/code.js]: slicing up abandoned pets? :dog:

**Comment by uttrasey** [comment]
:+1:

**Comment by uttrasey** [comment]
@trevorsenior Looks good, as you say it would be nice to have the form model update. You could do this with a ref to the form and manually firing it's `onchange` from adopt/abandon methods.

**Comment by uttrasey** [comment]
Looks good

**Pull request merged**